source/material_model/visco_plastic.cc

```diff
@@ -391,10 +413,26 @@ namespace aspect
           double density = 0.0;
           for (unsigned int j=0; j < volume_fractions.size(); ++j)
             {
+              double delta_crust_density = 0.0;
+              if (use_spcrust_density_change)
```

Comment:
Can you add a comment similar to the one for the `spcrust` viscosity above?

Reply:
done.